src/main/resources/lib/guillotine/generic/enum-types.js

```diff
@@ -24,6 +24,59 @@ function createEnumTypes(context) {
             'inline': 'inline'
         }
     });
+
+    context.types.dslQueryType = context.schemaGenerator.createEnumType({
+        name: context.uniqueName('DslQueryType'),
+        description: 'Query Dsl type.',
```

Comment:
`name - DslQueryType`
`description - Query Dsl type`

but 

`name - DslOperatorType`
`description - Dsl Operator Type`

It's better to have the same order everywhere

Reply:
Done